public/themes/dark-2.css

```diff
@@ -1,18 +1,26 @@
 .nylas-composer {
```

Comment:
why are these files in public/themes?

Reply:
not sure, this might've been where we were putting themes at one point